### PR TITLE
[BEAM-6850] Use HadoopFormatIOIT for performance/integration testing

### PR DESCRIPTION
**Pull request by aromanenko-dev**
Use `HadoopFormatIOIT` for performance/integration testing instead of deprecated `HadoopInputFormatIOIT`.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [x] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python3_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) <br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_PVR_Flink_Cron/lastCompletedBuild/) | --- | --- | ---

Pre-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

--- |Java | Python | Go | Website
--- | --- | --- | --- | ---
Non-portable | [Build Status](https://builds.apache.org/job/beam_PreCommit_Java_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Go_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Website_Cron/lastCompletedBuild/) 
Portable | --- | [Build Status](https://builds.apache.org/job/beam_PreCommit_Portable_Python_Cron/lastCompletedBuild/) | --- | ---

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

### CODE REVIEW

**Comment by aromanenko-dev** [comment]
Run Java HadoopFormatIO Performance Test

**Comment by lgajowy** [comment]
I'm on vacation until 5th may. Please reassign reviewer. Thanks :)

**Comment by aromanenko-dev** [comment]
@melap Could you take a look? I'm also wondering how to make new command `Run Java HadoopFormatIO Performance Test` working and run Jenkins job

**Comment by aromanenko-dev** [comment]
@kkucharc @mwalenia Could you take a look? I'm also wondering how to make new command `Run Java HadoopFormatIO Performance Test` working and run corresponding Jenkins job

**Comment by iemejia** [comment]
@chamikaramj can you please take a look at this one, it seems the polish crew are in holidays this week.

File: website/src/documentation/io/testing.md [hunk 1]
```diff
@@ -274,7 +274,7 @@ Since `performanceTest` task involved running PerfkitBenchmarker, we can't use i
 Example usage on Cloud Dataflow runner: 
 
 ```
-./gradlew integrationTest -p sdks/java/io/hadoop-input-format -DintegrationTestPipelineOptions='["--project=GOOGLE_CLOUD_PROJECT", "--tempRoot=GOOGLE_STORAGE_BUCKET", "--numberOfRecords=1000", "--postgresPort=5432", "--postgresServerName=SERVER_NAME", "--postgresUsername=postgres", "--postgresPassword=PASSWORD", "--postgresDatabaseName=postgres", "--postgresSsl=false", "--runner=TestDataflowRunner"]' -DintegrationTestRunner=dataflow --tests=org.apache.beam.sdk.io.hadoop.inputformat.HadoopInputFormatIOIT
+./gradlew integrationTest -p sdks/java/io/hadoop-format -DintegrationTestPipelineOptions='["--project=GOOGLE_CLOUD_PROJECT", "--tempRoot=GOOGLE_STORAGE_BUCKET", "--numberOfRecords=1000", "--postgresPort=5432", "--postgresServerName=SERVER_NAME", "--postgresUsername=postgres", "--postgresPassword=PASSWORD", "--postgresDatabaseName=postgres", "--postgresSsl=false", "--runner=TestDataflowRunner"]' -DintegrationTestRunner=dataflow --tests=org.apache.beam.sdk.io.hadoop.format.HadoopFormatIOIT
```
Comment (chamikaramj) [inline website/src/documentation/io/testing.md]: Please confirm that this works by running manually.

**Comment by chamikaramj** [comment]
Run Seed Job

**Comment by chamikaramj** [comment]
Run Java HadoopFormatIO Performance Test

**Comment by chamikaramj** [comment]
LGTM. Thanks.

**Comment by mwalenia** [comment]
@aromanenko-dev could you add metric gathering to `HadoopFormatIOIT`? It's included in `HadoopInputFormatIOIT`. Lack of metrics will disable reporting on dashboards, it would be good to keep the functionality intact.

File: website/src/documentation/io/testing.md (hunk 1 above)
Comment (aromanenko-dev) [reply website/src/documentation/io/testing.md]: I can confirm that It works fine with SparkRunner and local Postgres

**Comment by aromanenko-dev** [comment]
Run Java HadoopFormatIO Performance Test

**Comment by aromanenko-dev** [comment]
@mwalenia Yes, I copied these metric gathering into `HadoopFormatIOIT ` as well, thanks

**Comment by aromanenko-dev** [comment]
Run Seed Job

**Comment by aromanenko-dev** [comment]
Run Java HadoopFormatIO Performance Test

**Comment by lgajowy** [comment]
Cool - I updated the metrics plot for that: https://s.apache.org/io-test-dashboards (at the bottom). Thanks for doing this.

**Pull request merged**